benchmark/browser/webpack.config.js

```diff
@@ -16,7 +21,11 @@ module.exports = {
         test: /\.(js|ts|tsx)$/,
         exclude: /node_modules/,
         loader: 'babel-loader',
-        options: { cacheDirectory: true, cwd: workspaceRoot },
+        options: {
+          cacheDirectory: true,
+          configFile: path.join(workspaceRoot, 'babel.config.js'),
```

Comment:
Switched from `cwd` to an explicit `configFile`. `cwd` did mislead me because I assumed it applies to every relative filepath in the babel config. However, it only applies to the programmatic options of `babel-loader`:

> The working directory that all paths in the programmatic options will be resolved relative to.

-- https://babeljs.io/docs/en/options#cwd